src/collection/collectionwatcher.cpp

```diff
@@ -897,6 +952,25 @@ void CollectionWatcher::AddChangedSong(const QString &file, const Song &matching
 
 }
 
+#ifdef HAVE_EBUR128
+void CollectionWatcher::PerformEBUR128Analysis(SongList &songs, ScanTransaction *t) {
+
+  if (!song_ebur128_loudness_analysis_) return;
+
+  // FIXME: this should probably be done on thread pool.
```

Comment:
CollectionWatcher is running it a dedicated thread, running the analysis is in parallel will probably be slower because of disk reads, I've tested this with taglib and chromaprint. So I think you can just remove this comment.

Reply:
Err, i don't believe that to be true. `taglib` only reads metadata (a very small chunk of file),
`chromaprint` only takes first minute of the file, but "here" we process each song of the file.
Unless the kernel is absolutely unreasonable, the reads **will** be cached,
and the rest (per-song processing) is nicely parallelizable.